execution/executor-benchmark/src/gen_executor.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright © Aptos Foundation
+// Parts of the project are originally copyright © Meta Platforms, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{time::Duration, sync::atomic::AtomicUsize};
+use aptos_types::transaction::{SignedTransaction, Transaction};
+use aptos_types::account_address::AccountAddress;
+use aptos_transaction_generator_lib::TransactionExecutor as GenInitTransactionExecutor;
```

Comment:
What does  "gen init" mean in this context? 😂

Reply:
TransactionExecutor for transaction generator initialization. I didn't think of naming conflict when I was naming it that, maybe  aptos_transaction_generator_lib::TransactionExecutor should be renamed to something like ClientTransactionExecutor or ReliableTransactionSubmitter